### PR TITLE
Docker will need the config file at toplevel

### PR DESCRIPTION
**Pull request by jthoma**
I was attempting with the docker build, docker-compose up -d and the container were continuously restarting. Checking the logs showed the entry point was restarting with "rageshake.yaml" not found. I just added the docker-compose file also, since I find that easier than remembering the commands.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,6 +13,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -installsuffix cgo -ldflag
 FROM scratch
 COPY --from=builder /build/rageshake /rageshake
 COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=builder /build/rageshake.sample.yaml /rageshake.yaml
```
Comment (richvdh) [inline Dockerfile:16]: I'm not convinced this makes much sense. The sample config file isn't ready for use without customisation.

**Review by richvdh — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (jthoma) [reply Dockerfile:16]: Yes I do agree.. but no reference as to that it should be renamed to rageshake.yaml is anywhere in the install guide or so. Hence I just made the customisations and left the file as is.

Comment (jthoma) [reply Dockerfile:16]: But concise information is given that the customisations should be done in the rageshake.sample.yaml

**Review by jthoma — COMMENTED** [review]
Made some changes, such that the docker can expect a customized file. Complimentary addition is done to the README.md also.

**Review by richvdh — CHANGES_REQUESTED** [review]
This doesn't appear to work:

```
$ docker build .
...
Step 13/16 : COPY --from=builder /build/rageshake.yaml /rageshake.yaml
COPY failed: stat build/rageshake.yaml: file does not exist
```

I don't really think it makes sense to include a `rageshake.yaml` file inside the docker image, since everybody will need their own version. 

As noted in #32, the intention is that you supply your own `rageshake.yaml` with a `-v` option. If you'd like to update the documentation to make that clear, that would be welcome.

**Comment by richvdh** [comment]
I'm going to close this, as it seems to be abandoned.

**Pull request closed without merging**